OracleTuxedo/OracleTSAM/dockerfiles/12.2.2/Dockerfile

```diff
@@ -0,0 +1,130 @@
+# LICENSE CDDL 1.0 + GPL 2.0
+#
+# Copyright (c) 1996-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# (1) Oracle Tuxedo System and Applications Monitor (TSAM) Plus 12cR2 (12.2.2) GA Installer: tsam122200_64_Linux_x86.zip
+#     Download from http://www.oracle.com/technetwork/middleware/tuxedo/downloads/index.html
+#
+# (2) Oracle TSAM Plus 12.2.2 Rolling Patch 003: p25448043_12220_Linux-x86-64.zip
+#     Download from https://updates.oracle.com/Orion/Services/download/p25448043_12220_Linux-x86-64.zip?aru=20968945&patch_file=p25448043_12220_Linux-x86-64.zip
+#
+# (3) Oracle Database Instant Client: instantclient-basiclite-linux.x64-12.2.0.1.0.zip & instantclient-sqlplus-linux.x64-12.2.0.1.0.zip
```

Comment:
You should not use the zip files. Rather, switch to using the Instant Client RPMs on Oracle Linux